src/task24/task24.js

```diff
@@ -0,0 +1,13 @@
+function animate(time) {
+    let cat = document.getElementById("cat");
```

Comment:
Just move the cat, hat, angle, and lastTime it out.

Reply:
well, it now has this error
![image](https://user-images.githubusercontent.com/75950794/228480267-ce7c4fdd-1bcc-44e6-9061-3791035b8a00.png)
as if it doesn't see the variables
I suppose the matter is in the way I am calling this function